docs/NSR.md

```diff
@@ -220,7 +220,9 @@ CA Infrastructure MUST be in a Physically Secure Environment.
 
 ##### 1.2.2
 
-CA Infrastructure and Network Equipment MUST be implemented and configured to authenticate and encrypt connections:
+Connections to the CA Infrastructure MUST be authenticated and encrypted, except where documented that a formal specification prohibits or limits the use of authentication and/or encryption.
+
+Connections within the CA Infrastructure SHOULD be authenticated and encrypted.
```

Comment:
Cloud Services Discussion:

- Untrusted networks SHALL be encrypted and authn while Trusted networks SHOULD but is not required though moving to a zero trust model is what we're going towards. 
- Modern architecture using microservices SHOULD enable encrypting and authn. 
- Will have to have more discussions on what is in scope for CA Infrastructure as it is salient for this PR
- Should clarify Network Equipment definition later (either as part of this PR or another one)